### ISSUE TITLE
Store Audit Observe Permissions 

### ISSUE THREAD

**Issue by Toshbrown** [me-box/databox#272]
In version 0.0.8 of zestDB used in core-store, the audit observe permissions are tied to read permissions. So if an app wants to observe the audit logs for a whole store we currently have to grant READ on /* to that app. This is not very flexible and will need updating before the audit logs can be used by databox apps/drivers. 

I propose audit information is given a special endpoint (/audit) which could then be restricted by path if required. For example to observe logs for a datasource called ds1 we could grant READ on /audit/ds1/* to restrict the data leakage. Granting READ on /audit/* would give full access to the audit logs for all data sources in the store, but not the data in the store.

**Comment by jptmoore** [comment]
So a GET request which has an observe type of 'audit' should only be allowed through the prefix endpoint path of /audit ?

To observe audit on /kv/foo/* you need to have a valid token GET token for /audit/kv/foo/* ?

**Comment by Toshbrown** [comment]
Yes, that's what I'm suggesting. But if there is a better way to sperate GET (read) and Audit permissions I'm open to doing it another way.

**Comment by jptmoore** [comment]
I think it is the most straight forward way to implement this. I will have a go.
